src/app/containers/SignupPage/index.js

```diff
@@ -19,6 +19,11 @@ import { API_PROGRESS } from '../../../shared/constants/index';
 import { getErrors, sanitize, hasBlockingError, signupWithData } from './utils';
 
 function SignupPage({ history }) {
+  if (__IS_CONTEST_READY__) {
```

Comment:
it's just a temporary that I put to prevent user enter to the Enter route when we launch, this variable will be removed soon due to API getTime from BE will let us have a dynamically check